hive/src/main/java/org/apache/iceberg/hive/HiveCatalog.java

```diff
@@ -329,7 +355,13 @@ private boolean alterHiveDataBase(Namespace namespace,  Database database) {
 
     try {
       Database database = clients.run(client -> client.getDatabase(namespace.level(0)));
-      return convertToMetadata(database);
+      Map<String, String> metadata = convertToMetadata(database);
+      if (LOG.isDebugEnabled()) {
+        LOG.debug("For namespace: {} loaded metadata: {}",
+            namespace,
+            Joiner.on(",").withKeyValueSeparator("=").join(metadata));
+      }
```

Comment:
Same here. We should not be logging potentially sensitive properties.

Reply:
Removed